src/index.js

```diff
@@ -1,29 +1,15 @@
-/**
- * @file Exposes the agent and available features in multiple forms.
- */
+export { CustomAgent } from './custom-agent'
+export { BrowserAgent } from './browser-agent'
+export { WorkerAgent } from './worker-agent'
```

Comment:
Can we not have these onion layers whose only purpose is to export another file of same name in the `loaders` folder? Rather just import from loader/___ directly.